scripts/run_pytorch.sh

```diff
@@ -6,12 +6,14 @@ conda activate base
 ALL_FILE=$(find *.md ! -name README.md)
 TEMP_PY="temp.py"
 GANs="facebookresearch_pytorch-gan-zoo"
+CUDAs='nvidia'
+
 
 for f in $ALL_FILE
 do
   echo "Running pytorch example in $f"
-  # FIXME: GAN models checkpoints are on cuda.
-  if [[ $f = $GANs* ]]; then
+  # FIXME: GAN and NVIDIA models checkpoints are on cuda.
+  if [[ $f = $GANs* ]] || [[ $f = $CUDAs* ]]; then
```

Comment:
Adding a cuda checkpoint will require user have a cuda pytorch installed to load it. Would it possible to convert them to cpu in hubconf entrypoints and relax it here?

Reply:
considering this is NVIDIA, i think it's okay

Reply:
thx. I had to revive the above condition in run_pytorch.sh to pass your CI